project/query.py

```diff
@@ -0,0 +1,33 @@
+"""Query class."""
+
+import os
+import tweepy
+
+class Query(object): # pylint: disable=too-few-public-methods
+    """Class to query multiple API's."""
+    def __init__(self):
+        # Authenticates multiple API's
+
+        # Twitter authentification process
+        twit_consumer_key = os.environ.get('TWITTER_CONSUMER_KEY')
+        twit_consumer_secret = os.environ.get('TWITTER_CONSUMER_SECRET')
+        twit_access_token = os.environ.get('TWITTER_ACCESS_TOKEN')
+        twit_access_token_secret = os.environ.get('TWITTER_ACCESS_TOKEN_SECRET')
+        twitter_auth = tweepy.OAuthHandler(twit_consumer_key,
+                                           twit_consumer_secret)
+        twitter_auth.set_access_token(twit_access_token,
+                                      twit_access_token_secret)
+        self.twitter_api = tweepy.API(twitter_auth)
+
+
+    def twitter_search(self, query, num=100):
```

Comment:
I imagine we would have a query() function that queries every api/scraper. That way the caller doesn't need to worry about which api/scrapers are available. The query function can return a dictionary mapping API name to [tweet messages].

So right now it would return {'Twitter': [tweet1, tweet2, ...]}